runtime/nls/shrc/j9shr.nls

```diff
@@ -6929,3 +6929,17 @@ J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.explanation=Th
 J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.system_action=The JVM continues.
 J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.user_response=Contact your service representative.
 # END NON-TRANSLATABLE
+
+J9NLS_SHRC_OSCACHE_SEMAPHORE_OPFAILED_CONTROL_FILE_LOCK_FAILED_V1=An error has occurred while opening semaphore. Control file could not be open or locked. Please ensure the current user has right permissions on the cache directory and control file.
```

Comment:
I don't think new messages are required, you can just modify the existing ones to add the additional info.